sig-scalability/throughput_burts_slo.md

```diff
@@ -0,0 +1,23 @@
+# SLO: Kubernetes cluster of size at least X is able to start Y Pods in Z minutes
+
```

Comment:
I think that having `Z` instead of real number means exactly that, but I added a disclaimer as well.